src/PageNotFound/PageNotFound.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Another great idea to add this for error handling when the user goes to a bad url!